composer.json

```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": ">=5.3.3",
-        "friendsofsymfony/http-cache": "~1.2",
+        "friendsofsymfony/http-cache": "~1.3",
```

Comment:
Needs https://github.com/FriendsOfSymfony/FOSHttpCache/pull/190 so tag 1.3.2 and change this to `~1.3,>=1.3.2`